event_processor/event_processor/base/spider_base.py

```diff
@@ -45,4 +45,8 @@ def create_time_data(self, **kwargs):
                 raise ValueError(f'{self.organization}: Time selectors returned data of differing lengths')
         return [{key: value[i] for key, value in kwargs.items()} for i in range(count)]
 
+    def item_filter(self, item):
+        """Specify a filter for event items. This item should return True if 
+           the item should be include in the results and False if not. """
+        return True  
```

Comment:
This needs to be in aggregator_base because API-based spiders don't inherit from this class and will throw an error when this is called in the pipelines